connector-runtime/connector-runtime-spring/src/main/java/io/camunda/connector/runtime/inbound/controller/InboundConnectorRestController.java

```diff
@@ -83,7 +83,7 @@ private List<ActiveInboundConnectorResponse> getActiveInboundConnectors(
 
   private Map<String, Object> getData(ActiveExecutableResponse connector) {
     Map<String, Object> data = Map.of();
-    if (WebhookConnectorExecutable.class.equals(connector.executableClass())) {
+    if (WebhookConnectorExecutable.class.isAssignableFrom(connector.executableClass())) {
```

Comment:
Could you write a quick unit test that would catch the issue you discuss in PR description?

Reply:
Sure